subset/security/tlstest/src/main/java/Certs.java

```diff
@@ -45,7 +45,7 @@ public void get_certificate() {
           } catch (CertificateExpiredException cee) {
             certificateReport += "\nCertificate is expired";
```

Comment:
better to consistently add '\n' at the end of each one -- doing it this way is inconsistent and a bit weird :-).